core/src/scss/utilities/mixins/link/_link-icon.scss

```diff
@@ -1,21 +1,22 @@
 @charset "UTF-8";
 
 //
-// @link-icon($link-icon, $width: 0.65em, $animate: true)
+// @link-icon($icon, $width: 0.65em, $vertical: null, $animate: true)
 //
 // Display an icon after a link.
 //
 // $icon {string} - Basename of link icon .svg file (without the .svg suffix).
 // $width {string} - Width of icon including unit, e.g. 0.75em, 24px.
+// $vertical {string] - Adjustment of vertical position of icon, e.g., '2px' moves icon up 2 pixels, '-0.3em' moves it down 0.3 em.
 // $animate {boolean} - Whether or not to animate on hover.
 //
 // Style guide: Mixins.Link.link-icon
 //
-@mixin link-icon($icon, $width: 0.65em, $animate: true) {
+@mixin link-icon($icon, $width: 0.65em, $vertical: null, $animate: true) {
```

Comment:
Technically, changing the order of the parameter input is changing API so in order to not break API, you will need to add it to the end of the mixin parameters. It is much cleaner where it is right now, so if you would like to put this in as a PR to `v6` I'll be ok with where it is. 

Also, shouldn't the default value for the vertical alignment be `-1px` as that was the original value?

Reply:
Sounds good, @sherakama, and thanks for reviewing. When is v6 coming out? Is it after Decanterpalooza 3 this Dec? If so, then it's totally fine that we put this in v6. I just wanted to do this before we work on those new card/hero variants so that the icons are in the right place for those CTA links.

I actually found that a few of those icons don't need any vertical adjustments/margin-bottom so I'd rather have the default be `null` instead. Some of those with the `-1px` value look slightly misaligned.

Reply:
Actually, @sherakama, now that I think about it, I'm going to change the $animate parameter since Kerri asked for a few different animation. In that case, maybe I'll keep the original parameter order 🤔

Reply:
Yeah, V6 is planned for release after the next Palooza sprint which is in December. We will likely release a new version in January as we usually have a few items to wrap up after.